src/MailCollector.php

```diff
@@ -2276,6 +2276,19 @@ private function extractValuesFromRefHeader(string $header): ?array
             return $values;
         }
 
+        // Wrong message-Id format sometimes generated by GLPI >= 10.0.0 < 10.0.9
+        // - GLPI-{$uuid}-{$itemtype}-{$items_id}
+        $pattern = '/'
+            . 'GLPI'
+            . '-(?<uuid>[a-z0-9]+)' //  uuid
+            . '-(?<itemtype>[a-z]+)' // itemtype
+            . '-(?<items_id>[0-9]+)' // items_id
+            . '/i';
+        if (preg_match($pattern, $header, $values) === 1) {
+            $values += $defaults;
+            return $values;
+        }
```

Comment:
The previous code of `src/NotificationEventMailing.php` will create a`In-Reply-To` header like this :

`In-Reply-To: <GLPI-SgPctNbGMrzag8bWv3T17mdPnRqq5bBrYaXFMXIV-Ticket-81>`

This format is not detected with the current regexes and thus the collector won't detect that the email come from another GLPI  (which we are checking in `Collector::isResponseToMessageSentByAnotherGlpi()`).

Reply:
The `In-Reply-To` we are checking here is not the one sent by GLPI, but the one sent by the client, and this one contains the same value as the `Message-Id` header of the e-mail that is answered.

Reply:
It might come directly from GLPI if two mail collectors try to contact each other, which as far as I'm understand in the point of the `Collector::isResponseToMessageSentByAnotherGlpi()` function, no ?

Reply:
`isResponseToMessageSentByAnotherGlpi` is here to filter messages that are a response to a notification sent by another GLPI instance.
It does not check messages that directly came from another GLPI instance. We probably have to clarify this.